stl/inc/xstring

```diff
@@ -3980,6 +3980,25 @@ public:
         }
     }
 
+#if _HAS_CXX23
+    template <class _Operation>
+    constexpr void resize_and_overwrite(_CRT_GUARDOVERFLOW const size_type _Newsize, _Operation _Op) {
+        if (_Mypair._Myval2._Myres < _Newsize) {
+            _Reallocate_grow_by(
+                _Newsize - size(), [](_Elem* const _New_ptr, const _Elem* const _Old_ptr, const size_type _Oldsize) {
+                    _Traits::copy(_New_ptr, _Old_ptr, _Oldsize + 1);
+                });
+        }
+
+        const auto _Resultsize = _STD move(_Op)(data(), _Newsize);
+#if _CONTAINER_DEBUG_LEVEL > 0
+        _STL_VERIFY(_Resultsize >= 0, "the returned size can't be smaller than 0");

```

Comment:
maybe `if constexpr (is_signed_v<decltype(_Resultsize)>) { ... }` to avoid warning about condition that always evaluates to the same for unsigned?

Reply:
It seems that there's no such warning since the tests would raise it.